src/test/groovy/org/gradle/wrapperupgrade/GradleWrapperUpgradePluginFuncTest.groovy

```diff
@@ -49,6 +49,7 @@ class GradleWrapperUpgradePluginFuncTest extends Specification {
                         dir = 'samples/gradle'
                         options {
                             allowPreRelease = ${allowPreRelease}
+                            labels = ["dependencies", "java"]
```

Comment:
Can you please add the same to `MavenWrapperUpgradePluginFuncTest`?

Reply:
Added here https://github.com/gradle/wrapper-upgrade-gradle-plugin/pull/263/commits/dd7422dd1d882fe94392e525d21a72d406bdd78b